src/Evolution/Systems/Ccz4/DerivLapse.hpp

```diff
@@ -39,4 +39,34 @@ tnsr::ij<DataType, Dim, Frame> grad_grad_lapse(
     const tnsr::i<DataType, Dim, Frame>& field_a,
     const tnsr::ij<DataType, Dim, Frame>& d_field_a) noexcept;
 /// @}
+
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the divergence of the lapse.
+ *
+ * \details Computes the divergence as:
+ * \f{align}
+ *     \nabla^i \nabla_i \alpha &= \phi^2 \tilde{\gamma^{ij}}
```

Comment:
You should have the `tilde` over the indices. That is `\tilde{\gamma}^{ij}` not `\tilde{\gamma^{ij}}`